### DISCUSSION TITLE
Cruiser 38 download crash...

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#433] [Q&A] [answered]
I have installed Cruiser 38 from the scratch. Opening the map menu, changing to OpenAndroMaps in order to download the first map the app crashes immediately... Trying to reopen the map menu (OpenAndroMaps is the selected source) the app crashes again... Please check it, I can't use Cruiser anymore... 😖

**Comment by devemux86** [comment]
@kds112 thanks for the report.

Do you mean a clean install? What Android?

I cannot reproduce it (also don't see anything yet in Google Play):
- Clean install of Cruiser
- Select menu "Offline maps"
- Select OpenAndroMaps
- Download e.g. Andorra works

Have you installed Cruiser 38 (Beta) all this time? Was it working properly?

Or do you make specific steps to download? Or specific profile options?

**Reply by kds112** [reply]
Clean Installation on Realme 8 with Android 13. Opening app, map menu, select OpenAndroMaps, downloading Elevate theme and then trying to open Europe folder and app crashes...

**Reply by kds112** [reply]
I had no problems with the betas, but I updated them, I did no clean Installation from the scratch...

**Reply by devemux86** [reply]
> downloading Elevate theme and then trying to open Europe folder

@kds112 thanks, this is the important information, with these steps I can reproduce and will fix it.

In the meantime, please **download first the map** and the Elevate theme is downloaded automatically.
This should work fine.

**Comment by Ciutnik** [comment]
The same thing happened to me. After updating the OAM map (it was the only map for me), the Cruiser forces close. It only works with the online map, but trying to turn on the offline map immediately terminates the application. I noticed that the folder with maps used (on the SD card) is empty, although the ZIP was probably downloaded, it did not seem to be unpacked.
~~Tomorrow I will add the .map file to the directory manually and see if it works.~~

Edit: I manually put the OAM map into the maps directory - this time to the previously automatically created `openandromap` subdirectory. As before, the map update number was displayed. After updating everything works!
The only difference (apart from the location of the files in a subdirectory) is that this time I waited until the update icon disappeared in the map list and a message was displayed that there was no update. Previously, I seemed to have exited the update tab before the process was complete...

**Comment by MotoUKRider** [comment]
At the other end of the user experience: Samsung S20, Android 13 and yesterday I updated the France (Mapsforge) map (3GB) and POI without issue :smile:

**Comment by devemux86** [comment]
Thanks everyone for your reports! :+1: 

A new Cruiser update with the fix will be released today.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.39](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
